wc-admin.php

```diff
@@ -40,17 +40,17 @@ function wc_admin_plugins_notice() {
 		);
 	} else {
 		$message = sprintf(
-			/* translators: 1: URL of Gutenberg plugin, 2: URL of WooCommerce plugin */
-			__( 'The WooCommerce Admin feature plugin requires both <a href="%1$s">Gutenberg</a> and <a href="%2$s">WooCommerce</a> (>3.5) to be installed and active.', 'wc-admin' ),
-			'https://wordpress.org/plugins/gutenberg/',
+			/* translators: 1: URL of WordPress.org, 2: URL of WooCommerce plugin */
+			__( 'The WooCommerce Admin feature plugin requires both <a href="%1$s">WordPress</a> (>=5.0) and <a href="%2$s">WooCommerce</a> (>3.5) to be installed and active.', 'wc-admin' ),
```

Comment:
Should these be `target="_blank"`?

Reply:
No problem having no `target="_blank"`, but we could include buttons to install or update.
We also code change `>=` to `or greater` like WP does already:

```
The WooCommerce Admin feature plugin requires both <a href="%1$s">WordPress</a> 5.0 or greater and <a href="%2$s">WooCommerce</a> 3.5 or greater to be installed and active.
```